iron-form.html

```diff
@@ -135,27 +158,25 @@
       */
 
       attached: function() {
-        this._nodeObserver = Polymer.dom(this.$.content).observeNodes(
```

Comment:
Also, just double checked with @azakus and since this is a hybrid element, we can't reliably use `slotchanged` (since it has to work in Polymer 1, with the v0 spec, that does not have `slots`). As a result, we have to leave the current `observeNodes` approach in instead.